docs/develop/run-code-on-document-open.md

```diff
@@ -84,15 +87,18 @@ async function onChange(event) {
 
 ## Configure your add-in for no load behavior on document open
 
-The following code configures your add-in not to start when the document is opened. Instead, it will start when the user engages it in some way, such as choosing a ribbon button or opening the task pane.
+There may be scenarios when you want to turn off the "run on document open" behavior. The following code configures your add-in not to start when the document is opened. Instead, it will start when the user engages it in some way, such as choosing a ribbon button or opening the task pane. This code has no effect if the method has not previously been called, on the current document, with `Office.StartupBehavior.load` as the parameter.
```

Comment:
```suggestion
There may be scenarios when you want to turn off the "run on document open" behavior. The following code configures your add-in not to start when the document is opened. Instead, it will start when the user engages it in some way, such as choosing a ribbon button or opening the task pane. This code has no effect if the method hasn't previously been called on the current document, with `Office.StartupBehavior.load` as the parameter.
```